pxemgr/pxemanager.go

```diff
@@ -386,8 +383,7 @@ func (mgr *pxeManagerT) getNextInternalIP() net.IP {
 		currentIP = incIP(currentIP)
 	}
 
-	panic(errors.New("unable to get a free ip"))
-	return net.IP{}
+	//panic(errors.New("unable to get a free ip"))
```

Comment:
I cant really say what is happening there, it was `unreachable code` from go vet, we can remove it